GooglePlacesAutocompleteExample/GooglePlacesAutocompleteExampleTests/GooglePlacesAutocompleteExampleTests.swift

```diff
@@ -10,12 +10,16 @@ import Foundation
 import UIKit
 import XCTest
 import GooglePlacesAutocomplete
+import FBSnapshotTestCase
+import OHHTTPStubs
 
 class GooglePlacesAutocompleteTests: FBSnapshotTestCase, GooglePlacesAutocompleteDelegate {
   let gpaViewController = GooglePlacesAutocomplete(apiKey: "APIKEY")
   var expectation: XCTestExpectation!
 
-  func testGooglePlacesAutocomplete() {
+  // TODO: This test was failing because the snapshot was out-of-date. Let's do something better instead :)
```

Comment:
Groan.  You're right this looks like more trouble than it's worth.  I might pull in KIF to drive the UI tests then.